modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/AbstractEngineConfiguration.java

```diff
@@ -840,7 +842,16 @@ public int compare(EngineConfigurator configurator1, EngineConfigurator configur
 
         }
     }
-    
+
+    public void close() {
+        if (forceCloseMybatisConnectionPool
+                && dataSource instanceof PooledDataSource) {
+
+            // ACT-233: connection pool of Ibatis is not properely initialized if this is not called!
```

Comment:
@PascalSchumacher   I think this comment is not wrong. 
You can look at the code below
AbstractEngineConfiguration.java

 protected void initDataSource() {
           ...
            if (dataSource instanceof PooledDataSource) {
                // ACT-233: connection pool of Ibatis is not properly
                // initialized if this is not called!
                ((PooledDataSource) dataSource).forceCloseAll();
            }
        }

        if (databaseType == null) {
            initDatabaseType();
        }
    }

Reply:
I not sure why `PooledDataSource#forceCloseAll` is called during initialization, maybe this call was necessary in old versions of `PooledDataSource` to make the datasource use changed settings. I believe it can be removed see: https://github.com/flowable/flowable-engine/pull/1482

I may be wrong, but shouldn't an object be discarded (not used anymore) after its close method has been called? Therefore I think this comment is misleading.